recipes/libcurl/all/conanfile.py

```diff
@@ -64,6 +67,9 @@ class LibcurlConan(ConanFile):
         "with_c_ares": False,
         "with_proxy": True,
         "with_rtsp": True,
+        "with_crypto_auth": True,
+        "with_ntlm": True,
+        "with_ntlm_wb": True,
```

Comment:
Do these defaults keep "backwards" compatibility with earlier revisions of the recipe?

Reply:
Yes it does 🙂